src/ADS124S08.h

```diff
@@ -86,229 +86,229 @@
+#define ADS_IDAC2_A1 0x10
+#define ADS_IDAC2_A2 0x20
+#define ADS_IDAC2_A3 0x30
+#define ADS_IDAC2_A4 0x40
+#define ADS_IDAC2_A5 0x50
+#define ADS_IDAC2_A6 0x60
+#define ADS_IDAC2_A7 0x70
+#define ADS_IDAC2_A8 0x80
+#define ADS_IDAC2_A9 0x90
+#define ADS_IDAC2_A10 0xA0
+#define ADS_IDAC2_A11 0xB0
+#define ADS_IDAC2_AINCOM 0xC0
+#define ADS_IDAC2_OFF 0xF0
 /* Define IDAC1 Output */
-#define ADS_IDAC1_A0			0x00
-#define ADS_IDAC1_A1			0x01
-#define ADS_IDAC1_A2			0x02
-#define ADS_IDAC1_A3			0x03
-#define ADS_IDAC1_A4			0x04
-#define ADS_IDAC1_A5			0x05
-#define ADS_IDAC1_A6			0x06
-#define ADS_IDAC1_A7			0x07
-#define ADS_IDAC1_A8			0x08
-#define ADS_IDAC1_A9			0x09
-#define ADS_IDAC1_A10			0x0A
-#define ADS_IDAC1_A11			0x0B
-#define ADS_IDAC1_AINCOM		0x0C
-#define ADS_IDAC1_OFF			0x0F
+#define ADS_IDAC1_A0 0x00
+#define ADS_IDAC1_A1 0x01
+#define ADS_IDAC1_A2 0x02
+#define ADS_IDAC1_A3 0x03
+#define ADS_IDAC1_A4 0x04
+#define ADS_IDAC1_A5 0x05
+#define ADS_IDAC1_A6 0x06
+#define ADS_IDAC1_A7 0x07
+#define ADS_IDAC1_A8 0x08
+#define ADS_IDAC1_A9 0x09
+#define ADS_IDAC1_A10 0x0A
+#define ADS_IDAC1_A11 0x0B
+#define ADS_IDAC1_AINCOM 0x0C
+#define ADS_IDAC1_OFF 0x0F
 /* ADS124S08 Register 8 (VBIAS) Definition */
 /*   Bit 7   |   Bit 6   |   Bit 5   |   Bit 4   |   Bit 3   |   Bit 2   |   Bit 1   |   Bit 0
  *--------------------------------------------------------------------------------------------
  *  VB_LEVEL | 	VB_AINC  |  VB_AIN5  |  VB_AIN4  |  VB_AIN3  |  VB_AIN2  |  VB_AIN1  |  VB_AIN0
  *
  */
-#define ADS_VBIAS_LVL_DIV2		0x00
-#define ADS_VBIAS_LVL_DIV12		0x80
+#define ADS_VBIAS_LVL_DIV2 0x00
+#define ADS_VBIAS_LVL_DIV12 0x80
 /* Define VBIAS here */
-#define ADS_VB_AINC				0x40
-#define ADS_VB_AIN5				0x20
-#define ADS_VB_AIN4				0x10
-#define ADS_VB_AIN3				0x08
-#define ADS_VB_AIN2				0x04
-#define ADS_VB_AIN1				0x02
-#define ADS_VB_AIN0				0x01
+#define ADS_VB_AINC 0x40
+#define ADS_VB_AIN5 0x20
+#define ADS_VB_AIN4 0x10
+#define ADS_VB_AIN3 0x08
+#define ADS_VB_AIN2 0x04
+#define ADS_VB_AIN1 0x02
+#define ADS_VB_AIN0 0x01
 /* ADS124S08 Register 9 (SYS) Definition */
 /*   Bit 7   |   Bit 6   |   Bit 5   |   Bit 4   |   Bit 3   |   Bit 2   |   Bit 1   |   Bit 0
  *--------------------------------------------------------------------------------------------
  *			   SYS_MON[2:0]			 |	   CAL_SAMP[1:0]     |  TIMEOUT  | 	  CRC	 | SENDSTAT
  *
  */
-#define ADS_SYS_MON_OFF			0x00
-#define ADS_SYS_MON_SHORT		0x20
-#define ADS_SYS_MON_TEMP		0x40
-#define ADS_SYS_MON_ADIV4		0x60
-#define ADS_SYS_MON_DDIV4		0x80
-#define ADS_SYS_MON_BCS_2		0xA0
-#define ADS_SYS_MON_BCS_1		0xC0
-#define ADS_SYS_MON_BCS_10		0xE0
-#define ADS_CALSAMPLE_1			0x00
-#define ADS_CALSAMPLE_4			0x08
-#define ADS_CALSAMPLE_8			0x10
-#define ADS_CALSAMPLE_16		0x18
-#define ADS_TIMEOUT_DISABLE		0x00
-#define ADS_TIMEOUT_ENABLE		0x04
-#define ADS_CRC_DISABLE			0x00
-#define ADS_CRC_ENABLE			0x02
-#define ADS_SENDSTATUS_DISABLE	0x00
-#define ADS_SENDSTATUS_ENABLE	0x01

```

Comment:
again - none of these lines have been actually changed, but since you (or your editor) changed the spacing, this is seen as a change